app-i18n/mozc/mozc-2.23.2785.102.recipe

```diff
@@ -2,12 +2,12 @@ SUMMARY="Japanese input method editor"
 DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
 This is open source version."
 HOMEPAGE="https://github.com/google/mozc/"
-COPYRIGHT="2010-2016, Google Inc."
+COPYRIGHT="2010-2018, Google Inc."
 LICENSE="BSD (3-clause)"
-REVISION="2"
-fileVersion="v2.20.2677.102.6"
+REVISION="1"
+fileVersion="v2.23.2785.102.1"
 SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
```

Comment:
If you rename the recipe to `mozc-2.23.2785.102.1.recipe` then you can:
```diff
-fileVersion="v2.23.2785.102.1"
-SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
+SOURCE_URI="https://github.com/hanya/mozc/releases/download/v$portVersion/mozc-v$portVersion.tar.bz2"
```
BTW, if you do so, don't forget to also rename the patchset accordingly.

Important: I don't know if there is any downside to renaming the patchset, so please feel free to ignore this comment if needed.
(If there is no downside then I recommend doing so.)

Reply:
I've successively built mozc on x86_64. And it looks like it is working like a charm!

BTW, before building mozc, I made the changes suggested in my above message, renamed the recipe to `mozc-2.23.2785.102.1.recipe` and the patch to `mozc-2.23.2785.102.1-x86.patch`.

As expected, haikuporter and pkgman did not complain about the quite long `$portVersion`.
So it would be great if you could do follow the above suggestion. Thanks!